runserver.sh

```diff
@@ -10,25 +10,13 @@
 set -e
 
 standalone() {
-  mysql(){
-  if ! mysql.server status; then
-    mysql.server start
-  fi
-  }
-
-  # Run tasks to build the project for distribution.
-  server(){
   if [ "$1" == "ssl" ]; then
     echo '\033[0;32mStarting SSL Django server on port' $DJANGO_HTTP_PORT '...'
```

Comment:
not related to this PR, but if we want color output, I think we need to use `echo -e` or `printf`

Reply:
I've long wondered what those characters were for!